tests/select/test_drop_pv_generating_overnight.py

```diff
@@ -0,0 +1,53 @@
+from datetime import datetime
+from typing import Optional
+
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+from ocf_datapipes.select import DropNightPV
+from ocf_datapipes.transform.xarray import AssignDayNightStatus
+
+
+def test_drop_with_pvoutput_datapipe(pvoutput_datapipe):
+    before_dropping_pv_with_night_output = AssignDayNightStatus(pvoutput_datapipe)
+    after_dropping_pv_with_night_output = DropNightPV(before_dropping_pv_with_night_output)
+
+    data_before_drop = next(iter(before_dropping_pv_with_night_output))
+    data_after_drop = next(iter(after_dropping_pv_with_night_output))
+
+    # In 'pvoutput_datapipe', there are 41 or so systems and some of the systems have been dropped
+    # as they generate power over night.
+    assert len(data_before_drop.coords["pv_system_id"].values) != len(
```

Comment:
perhaps good to specify how many have been dropped, and then we can keep testing for that exact amount

Reply:
Updated the code with the number of systems remaining and number of systems that have been dropped!